src/calendar-list/index.tsx

```diff
@@ -138,18 +140,19 @@ class CalendarList extends Component<CalendarListProps, CalendarListState> {
     const rows = [];
     const texts = [];
     const date = parseDate(props.current) || new XDate();
+    const {pastScrollRange = 50, futureScrollRange = 50} = props;
```

Comment:
I see we're setting default prop value for pastScrollRange and futureScrollRange in many places in this file. 
I know it's for typescript cause TS doesn't understand `static defaultProps` anyway let's at least define a const for these default values at the top of the file.

Reply:
Done